src/secret/mod.rs

```diff
@@ -149,6 +149,8 @@ pub enum SecretManagerDto {
     LedgerNanoSimulator,
     /// Mnemonic
     Mnemonic(String),
+    /// Hex seed
+    HexSeed(String),
```

Comment:
I think I would have called it just `Seed` tbh but whatever

Reply:
The method is called from_hex_seed too so for consistency i did the same